### ISSUE TITLE
Xcode 6 Cocoa Touch Framework

### ISSUE THREAD

**Issue by shwetsolanki-lrm** [CocoaPods/Specs#12882]
Hey, I am created the podspec file for the Cocoa Touch Framework. I have used **"vendored_frameworks"**. Now when I try to install the podspec in my project, the Framework is added correctly. But when run the app, I get the following error.

**dyld: Library not loaded: @rpath/Some.framework/Some
  Referenced from: /private/var/mobile/Containers/Bundle/Application/45B49F35-31E2-49F6-8102-26FF43F0BB2E/AppLifecycle.app/AppLifecycle
  Reason: image not found**

I am sure this happening just because the framework is not added as **Embedded Binaries** and in the Project Target Settings. Could you tell me a way so that this is included in the podspec itself. And moreover, when I manually try to add the Some.framework, its not listed.

**Comment by shwetsolanki-lrm** [comment]
This is the spec 

https://github.com/shwetsolanki-lrm/TestFramework/blob/master/Test-iOS-SDK.podspec

**Comment by Keithbsmiley** [comment]
Sounds like some sort of framework configuration issue. That is the one and only way to ship frameworks via CocoaPods. You should ask this question on stackoverflow.

**Comment by shwetsolanki-lrm** [comment]
Okie, could you tell me how do I add my framework to the Projects **Copy Files** in **Build Phases**?

Check the image attached.

![screenshot 2015-01-03 13 29 29](https://cloud.githubusercontent.com/assets/9427574/5601911/c3aa306e-934c-11e4-9e85-091aaf06b63a.png)

**Comment by Keithbsmiley** [comment]
This is not how CocoaPods works. It automatically links them with the xcconfigs.

**Comment by shwetsolanki-lrm** [comment]
I would like you to check this https://medium.com/@syshen/create-an-ios-universal-framework-148eb130a46c

This post it explains how the Xcode 6 cocoa touch framework is created. I have followed the same steps. Check the Troubleshoot section. Its talking about the same problem which I have raised. Now, in the blog post, framework is included manually by Drag n Drop and thus the user have to make sure that Framework is also added to Embedded Binaries. Now I need automate this via CocoaPods, so that the framework is added to Embedded Binaries automatically.

I think you guys really need to investigate on this issue.
